src/resolvers-cassandra/Messages/queries.js

```diff
@@ -29,6 +35,49 @@ function byEdges(args, res) { // eslint-disable-line no-unused-vars
  * @returns {Promise.<Feature>}
  */
 function event(args, res) { // eslint-disable-line no-unused-vars
+  const eventById = 'SELECT * FROM fortis.events WHERE id = ?';
+
+  return new Promise((resolve, reject) => {
+    const eventId = args && args.messageId;
+    if (!eventId) {
+      reject('No event id to fetch specified');
```

Comment:
If you would like I believe you can also just do `return reject('No event id to fetch specified');` and it should still prevent the below lines from executing.